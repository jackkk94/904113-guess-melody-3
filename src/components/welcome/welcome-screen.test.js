import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeScreen from './welcome-screen';

const name = `Alex`;

it(`<WelcomeScreen /> should render Alex`, () => {
  const tree = renderer
    .create(<WelcomeScreen name = {name} onButtonClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

