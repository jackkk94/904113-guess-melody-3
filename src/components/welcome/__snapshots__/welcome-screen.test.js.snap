// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeScreen /> should render Alex 1`] = `
<div>
  Hello 
  Alex
   !
</div>
`;
